plugins/spring-boot/info/info.component.ts

```diff
@@ -0,0 +1,67 @@
+/// <reference path="info.service.ts"/>
+/// <reference path="info.ts"/>
+
+namespace SpringBoot {
+
+  export class InfoController {
+    readonly reloadDelay = 20000;
+    reloadTask: ng.IPromise<any>;
+    info: Info;
+    info_pair: Info
+
+    constructor(private $interval: ng.IIntervalService, private infoService: InfoService) {
+      'ngInject';
+    }
+
+    $onInit() {
+      this.loadDataPeriodically();
+    }
+
+    $onDestroy() {
+      this.cancelReloadTask();
+    }
+
+    loadDataPeriodically() {
+      this.loadData();
+      this.reloadTask = this.$interval(() => this.loadData(), this.reloadDelay);
+    }
+
+    loadData() {
+      this.infoService.getInfo()
+        .then(info => {
+          this.info = info
+          this.info_pair = _.toPairs(info)
```

Comment:
if we only use the `infoPair` in html we don't need `info` at all. I'd rename it as `info` and use it in html.